lib/AST/Decl.cpp

```diff
@@ -1864,13 +1875,13 @@ bool PatternBindingDecl::isDefaultInitializable(unsigned i) const {
 
   // If the outermost attached property wrapper vends an `init()`, use that
   // for default initialization.
+  if (isDefaultInitializableViaPropertyWrapper(i))
+    return true;
+
+  // If one of the attached wrappers is missing an initialValue
```

Comment:
Nit: should we be using `wrappedValue` terminology from now on? All the appearances of `initialValue` confused me at first because I didn't realize the name had changed over the course of review

Reply:
Yes, you're right. I copied this code from elsewhere and missed that this was out-of-date, thanks!

Reply:
Looks like there are a couple of APIs that also mention `initialValue`, so there's more cleanup to do here.